templates/git_client_side_hooks/pre-push

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+# Copyright 2020 Antrea Authors
```

Comment:
s/2020/2022/

Reply:
What?

Reply:
you should replace the year from 2020 to 2022.